lib/msf/util/host.rb

```diff
@@ -52,7 +49,6 @@ def self.normalize_host(host)
         # to try, just log it and return what we were given
         if !norm_host
           dlog("Host could not be normalized: #{host.inspect}")
```

Comment:
Makes sense, if `nil` was passed in as the argument it would have been returned. That doesn't change, this just adds a hostname that can't resolve to the list of conditions in which `nil` will be returned.